### PR TITLE
OH2-258 | Change the value of enum LaboratoryStatus in lowercase

### PR DESCRIPTION
**Pull request by ArnaudFonzam**
See OH2-258.

### CODE REVIEW

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 1]
```diff
@@ -76,13 +76,13 @@ public class LaboratoryController {
...
+	private static String draft = LaboratoryStatus.draft.toString();
 	
-	private static String open = LaboratoryStatus.OPEN.toString();
+	private static String open = LaboratoryStatus.open.toString();
 
-	private static String deleted = LaboratoryStatus.DELETED.toString();
+	private static String deleted = LaboratoryStatus.deleted.toString();
 	
-	private static String invalid = LaboratoryStatus.INVALID.toString();
+	private static String invalid = LaboratoryStatus.invalid.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:85]:
```suggestion
	private static String invalid = LaboratoryStatus.invalid.toString();
	
	private static String done = LaboratoryStatus.done.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 2]
```diff
@@ -185,7 +185,7 @@ public ResponseEntity<Boolean> newExamRequest(@RequestBody LaboratoryDTO laborat
 		labToInsert.setResult("");
 		labToInsert.setInOutPatient(laboratoryDTO.getInOutPatient().toString());
 		List<Laboratory> labList = laboratoryManager.getLaboratory(patient).stream()
-				.filter(e -> e.getStatus().equals(LaboratoryStatus.DRAFT.toString())).collect(Collectors.toList());
+				.filter(e -> e.getStatus().equals(LaboratoryStatus.draft.toString())).collect(Collectors.toList());
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:188]:
```suggestion
				.filter(e -> e.getStatus().equals(draft)).collect(Collectors.toList());
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 3]
```diff
@@ -303,7 +303,7 @@ public ResponseEntity<Boolean> updateLaboratory(@PathVariable Integer code,
 			labRows = new ArrayList<>(labRow);
 		}
 		if (!laboratoryDTO.getResult().isEmpty()) {
-			labToInsert.setStatus(LaboratoryStatus.DONE.toString());
+			labToInsert.setStatus(LaboratoryStatus.done.toString());
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:306]:
```suggestion
			labToInsert.setStatus(done);
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/lab/rest/LaboratoryController.java:85]: As `done` is a `static` shouldn't it be:
<pre>
private static final String DONE = LaboratoryStatus.done.toString();
</pre>

Comment (mwithi) [reply src/main/java/org/isf/lab/rest/LaboratoryController.java:85]: yes, you are right, all of them

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 4]
```diff
@@ -76,13 +76,15 @@ public class LaboratoryController {
 	// TODO: to centralize
 	protected static final String DEFAULT_PAGE_SIZE = "80";
 	
-	private static String draft = LaboratoryStatus.DRAFT.toString();
+	private static String draft = LaboratoryStatus.draft.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:79]:
```suggestion
	private static String DRAFT = LaboratoryStatus.draft.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 5]
```diff
@@ -76,13 +76,15 @@ public class LaboratoryController {
...
 	
-	private static String open = LaboratoryStatus.OPEN.toString();
+	private static String open = LaboratoryStatus.open.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:81]:
```suggestion
	private static String OPEN = LaboratoryStatus.open.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 6]
```diff
@@ -76,13 +76,15 @@ public class LaboratoryController {
...
 
-	private static String deleted = LaboratoryStatus.DELETED.toString();
+	private static String deleted = LaboratoryStatus.deleted.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:83]:
```suggestion
	private static String DELETED = LaboratoryStatus.deleted.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 7]
```diff
@@ -76,13 +76,15 @@ public class LaboratoryController {
...
 	
-	private static String invalid = LaboratoryStatus.INVALID.toString();
+	private static String invalid = LaboratoryStatus.invalid.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:85]:
```suggestion
	private static String INVALID = LaboratoryStatus.invalid.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java [hunk 8]
```diff
@@ -76,13 +76,15 @@ public class LaboratoryController {
...
+	
+	private static String done = LaboratoryStatus.done.toString();
```
Comment (mwithi) [inline src/main/java/org/isf/lab/rest/LaboratoryController.java:87]:
```suggestion
	private static String DONE = LaboratoryStatus.done.toString();
```

File: src/main/java/org/isf/lab/rest/LaboratoryController.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/lab/rest/LaboratoryController.java]: @ArnaudFonzam I've applied myself the suggested changes, please pull and align the rest. Thanks!

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**